terraform/modules/iam_role_policy/cloudwatch/policy.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents",
+        "logs:DescribeLogStreams",
+        "cloudwatch:ListMetrics",
```

Comment:
It looks like I also decided to depend on this being a thing in order to fail fast if the AWS creds are bad: https://github.com/18F/cg-awslogs-boshrelease/blame/master/jobs/awslogs/templates/bin/awslogsd.sh.erb#L23 but I guess we can drop that whole check (or find a better way to do it) once we aren't relying on creds.